<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowEditor</class>
 <widget class="QMainWindow" name="MainWindowEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="FilePart">
     <attribute name="title">
      <string>File</string>
     </attribute>
     <widget class="QLineEdit" name="inputThing">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="enterBtn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>110</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="EditPart">
     <attribute name="title">
      <string>Edit</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>6</y>
        <width>371</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>51</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
